File: linked-list/383.js
var Solution = function(head) {
    this.arr = []
    let temp = head
    while(temp){
        this.arr.push(temp.val)
        temp = temp.next
    }
};

Solution.prototype.getRandom = function() {
    const n = this.arr.length
    const randomIndex = Math.floor(Math.random() * n)
    return this.arr[randomIndex]
};

File: linked-list/383.py
import random
class Solution:

    def __init__(self, head: Optional[ListNode]):
        self.arr = []
        temp = head
        while temp:
            self.arr.append(temp.val)
            temp = temp.next

    def getRandom(self) -> int:
        n = len(self.arr)
        random_index = random.randint(0, n - 1)
        return self.arr[random_index]
